app/controllers/provider_interface/provider_users_invitations_controller.rb

```diff
@@ -106,13 +106,12 @@ def permissions_params
         .permit(provider_permissions: {})
     end
 
-    def require_feature_flag
+    def require_feature_flag!
       render_404 unless FeatureFlag.active?(:providers_can_manage_users_and_permissions)
     end
 
-    def redirect_unless_permitted_to_manage_users
-      can_manage_users = ProviderPermissions.exists?(provider_user: current_provider_user, manage_users: true)
-      render_404 unless can_manage_users
+    def require_manage_user_permission!
+      render_404 unless current_provider_user.authorisation.can_manage_users?
```

Comment:
I like seeing `.authorisation.` in the chain here rather than not due to some magic delegation